Datadog/Example/Debugging/DebugRUMSessionViewController.swift

```diff
@@ -171,7 +171,7 @@ internal struct DebugRUMSessionView: View {
                     .listRowInsets(EdgeInsets())
                     .padding(4)
             }
-            .listStyle(.plain)
+            .listStyle(PlainListStyle())
```

Comment:
curious: is the rest of the PR necessary for the fix?

Reply:
Yes, in some CI logs from Xcode 12 I can also see:
```
❌  /Users/vagrant/git/Datadog/Example/Debugging/DebugRUMSessionViewController.swift:47:23: variable used within its own initial value
        let viewKey = viewKey
                      ^
```
Some other failed with:
```
▸ Running script '⚙️ Run linter'
Automatic retry reason found in log: Early unexpected exit, operation never finished bootstrapping - no restart will be attempted
isAutomaticRetryOnReason=false, no more retry, stopping the test!
```
and I don't yet have a clue on what's wrong, but it seems unrelated to #669 changes.

Although dismissing Xcode's "upgrade workspace to recommended changes" warning won't fix it I played a boy-scout to cleanup our logs.